buildSrc/settings.gradle.kts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (C) 2022 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+dependencyResolutionManagement {
+  versionCatalogs { create("libs") { from(files("../gradle/libs.versions.toml")) } }
```

Comment:
Why put the versions file under `gradle`? I guess it's important enough to be a sibling to the build scripts :thinking:

Reply:
It's expected to be under `gradle/` (see [here](https://docs.gradle.org/current/userguide/platforms.html#sub:conventional-dependencies-toml))

Reply:
TIL - thx :+1:

Reply:
This still looks odd after reading the docs. We load the file explicitly, so I'd guess it could be located anywhere :thinking: 

Does it have to be under `gradle` for `renovate`?

Reply:
Nope - that's only for buildSrc (because that's always special :( )

Reply:
But this way the version declarations can be shared everywhere w/o having the "temporary file hacks"